types/src/v0/impls/stake_table.rs

```diff
@@ -147,24 +125,62 @@ impl StakeTableDelta {
         }
     }
 }
+/// Holds Stake table and da stake
+#[derive(Clone, Debug)]
+struct StaticCommittee {
+    /// The nodes eligible for leadership.
+    /// NOTE: This is currently a hack because the DA leader needs to be the quorum
+    /// leader but without voting rights.
+    eligible_leaders: Vec<StakeTableEntry<PubKey>>,
+
+    /// TODO: add comment
+    quorum: HashMap<PubKey, StakeTableEntry<PubKey>>,
```

Comment:
Why does order matter here?

Reply:
We found out that all hotshot nodes need to always compute the same stake table. Since the order of HashMap is not preserved for clones, etc. that won't work.